### PR TITLE
Rename `Geneva.UnitTests` to `Geneva.Tests`

### PR DESCRIPTION
**Pull request by mic-max**
Guide I followed: https://medium.com/c-sharp-progarmming/safely-rename-a-project-folder-visual-studio-f3c6bd4d0bd6

### CODE REVIEW

File: test/OpenTelemetry.Exporter.Geneva.Tests/OpenTelemetry.Exporter.Geneva.Tests.csproj [hunk 1]
```diff
@@ -8,6 +8,10 @@
     <NoWarn>$(NoWarn),SA1633,SA1311,SA1312,SA1313,SA1123,SA1202</NoWarn>
   </PropertyGroup>
 
+  <ItemGroup>
+    <AdditionalFiles Remove="C:\Git\opentelemetry-dotnet-contrib\build\/stylecop.json" />
```
Comment (cijothomas) [inline test/OpenTelemetry.Exporter.Geneva.Tests/OpenTelemetry.Exporter.Geneva.Tests.csproj:12]: ?

Comment (mic-max) [reply test/OpenTelemetry.Exporter.Geneva.Tests/OpenTelemetry.Exporter.Geneva.Tests.csproj]: Oops, undid

File: .github/workflows/windows-ci.yml [hunk 2]
```diff
@@ -28,4 +28,4 @@ jobs:
       run: dotnet build --configuration Release --no-restore
 
     - name: Test ${{ matrix.version }}
-      run: dotnet test **\bin\**\${{ matrix.version }}\*Tests.dll --configuration Release --no-build --logger:"console;verbosity=detailed"
+      run: dotnet test **\bin\**\${{ matrix.version }}\*Tests.dll --configuration Release --no-build --logger:"console;verbosity=detailed"  --filter "Platform=Any|Platform=Windows"
```
Comment (cijothomas) [inline .github/workflows/windows-ci.yml:31]: for now, we can proceed.
 But as a general rule, the common workflows should not be required to be modified to meet the needs of individual components. 
Consider using an approach similar to the below, to detect OS and skip the tests.
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/test/OpenTelemetry.Instrumentation.W3cTraceContext.Tests/W3CTraceContextTests.cs#L40

Please open an issue to track this, and we can unblock this PR.

**Review by cijothomas — APPROVED** [review]

File: .github/workflows/windows-ci.yml (hunk 2 above)
Comment (mic-max) [reply .github/workflows/windows-ci.yml:31]: Okay! https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/281

**Review by CodeBlanch — APPROVED** [review]
LGTM

**Pull request merged**